### PR TITLE
Add Parca project to applications

### PR DESCRIPTION
**Pull request by brancz**

### CODE REVIEW

**Review by vannyle — APPROVED** [review]

File: src/pages/applications.js [hunk 1]
```diff
@@ -126,6 +127,21 @@ const majorProjects = [
...
+    logo: parcaLogo,
+    title: 'Continuous Profiling Platform',
+    description: `Track memory, cpu, I/O bottlenecks broken down by method
+    name, class name, and  line number over time. Without complex overhead, in
+    any language or framework. Using Parca's UI the data can be globally
+    explored and analyzed using various visualizations to quickly and
+    efficiently identify bottlenecks in code.`,
+    urls: [
+      {label: 'GitHub', url: 'https://github.com/parca-dev/parca'},
+      {label: 'Website', url: 'https://parca.dev'},
```
Comment (borkmann) [inline src/pages/applications.js:142]: Hi @brancz thanks for the PR! Could you rebase and please also add to the description the relation of parca to eBPF? Presumably it's using eBPF (via rust? https://github.com/parca-dev/parca-agent) to gather all the data from the kernel. Also, please add it to the list of emerging projects given the stated requirements are not yet met at this point. Thanks a lot.

**Review by borkmann — CHANGES_REQUESTED** [review]

File: src/pages/applications.js (hunk 1 above)
Comment (borkmann) [reply src/pages/applications.js:142]: (ping @brancz )

Comment (brancz) [reply src/pages/applications.js:147]: Added some more on that topic. We dabbled with rust, but recently went back to writing ebpf programs in C. Not sure it makes sense to call that out here, but if you want to I can add it.

**Comment by xmulligan** [comment]
Thanks for adding @brancz 🐝 ! See you at eBPF Summit

**Pull request merged**